.github/workflows/ci.yml

```diff
@@ -203,7 +203,7 @@ jobs:
         id: build-docker-image
         if: steps.prepare-meta.outputs.has-image
         timeout-minutes: 120
-        run: cargo xtask build-docker-image -v "${TARGET}${SUB:+.$SUB}" ${{ matrix.verbose && '-v' || ''  }} --platform ${{ matrix.platform }}
+        run: cargo xtask build-docker-image -v "${TARGET}${SUB:+.$SUB}" ${{ matrix.verbose && '-v' || ''  }}${{ matrix.platform && '--platform ' }}${{ join(matrix.platform) || '' }}
```

Comment:
This change seems not necessary

Reply:
It's a very neat trick, if platforms are configured they will be used, otherwise it will be empty.